src/chart/tree/TreeSeries.js

```diff
@@ -139,6 +139,9 @@ export default SeriesModel.extend({
         // the layout of the tree, two value can be selected, 'orthogonal' or 'radial'
         layout: 'orthogonal',
 
+        // value can be 'polyline'
+        edgeLayout: 'curve',
```

Comment:
I think `edgeShape` is better. `edgeLayout` is more like the `position` of edge

Reply:
yeah, good suggestion!